### DISCUSSION TITLE
Missing example sentence in 点が辛い entry

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#58] [Errors and Corrections]
I see for words like 怠け者 we have the example sentence: 彼は怠け者に対して点が辛い。

I think this sentence can be used as a sample sentence for the 点が辛い entry as well, which has no example sentences as of now.

**Comment by stephenmk** [comment]
Thanks, that sentence has now been indexed to 点が辛い as well.
